backend/src/main/java/com/huellapositiva/application/dto/RegisterESALMemberRequestDto.java

```diff
@@ -0,0 +1,55 @@
+package com.huellapositiva.application.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import javax.validation.constraints.*;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class RegisterESALMemberRequestDto {
+
+    @Schema(
+            description = "Name of Contact Person",
+            example = "Guajiro Ermenegildo"
+    )
+    @NotBlank
+    @Pattern(regexp = "^[a-zA-Z ]+$")
+    private final String name;
+
+    @Schema(
+            description = "Surname of volunteer",
+            example = "Sánchez Inmaculado"
+    )
+    @NotBlank
+    @Pattern(regexp = "^[a-zA-Z ]+$")
+    private final String surname;
+
+    @Schema(
+            description = "User's telephone. TIP: The international preffix can be +X, +XX, +XXX",
+            example = "+850 12419287524"
+    )
+    @NotBlank
+    private final String phoneNumber;
+
+    @Schema(
+            description = "Organization email unique",
+            example = "huelladigital@soymamut.com"
+    )
+    @NotBlank
+    @Email
+    private final String email;
+
+    @Schema(
+            description = "Organization password",
+            example = "albaricoque29"
```

Comment:
The example password should be a strong one. We would like to promote security best practices across our users.

Reply:
Wifi pass: )OAx7H6$d'2]5F5R,}i%